types/react-fontawesome/react-fontawesome-tests.tsx

```diff
@@ -14,3 +14,14 @@ class TestComponent extends React.Component {
     );
   }
 }
+
+class CSSTestComponent extends React.Component {
+  render() {
+    return (
+      <FontAwesome
+        name="rocket"
+        style={{backgroundColor: 'red', color: 'blue'}}
+      />
+    );
+  }
+}
```

Comment:
I was on mac, and I saw that there's no new line here: https://github.com/gavingregory/DefinitelyTyped/blob/b4c8fdee9d12124e2f404c6e07399f56c471a15d/types/react-fontawesome/react-fontawesome-tests.tsx

It's OK if it's OK though 👍